code/game/objects/structures/watercloset.dm

```diff
@@ -445,6 +567,45 @@
 
 	if(!(istype(O)))
 		return
+	if(iswrench(O) && !istype(src, /obj/structure/sink/puddle))		//puddles laugh at your attempt to disconnect them. you can't remove a puddle from the ground with a wrench!
+		var/obj/item/weapon/wrench/W = O
+		if(anchored)
+			var/response = alert(user, "Would you like to wash [W] or disconnect [src]?", "Wrench", "Wash", "Disconnect")
+			if(response == "Disconnect")
+				user.visible_message("[user] is disconnecting [src].", "<span class='notice'>You are disconnecting [src]...</span>")
```

Comment:
Tenses and span classes are fucked up again.
`"<span class='notice'>[user] starts disconnecting [src]...</span>", "<span class='notice'>You start disconnecting [src]...</span>`